website/docs/r/dns_record.html.markdown

```diff
@@ -1,172 +1,185 @@
+* `domain_id` - (Required, integer) The ID for the domain associated with the resource record.
+* `expire` - (Optional, integer) The duration, expressed in seconds, that a secondary name server (or servers) holds a zone before it is no longer considered authoritative.
+* `host` - (Required, string) The host defined by a resource record. The `@` symbol denotes a wildcard.
+* `minimum_ttl` - (Optional, integer) The duration, expressed in seconds, that a domain's resource records are valid. This is also known as a minimum time to live (TTL), and can be overridden by an individual resource record's TTL.
+* `mx_priority` - (Optional, integer) The priority of the mail exchanger that delivers mail for a domain. This is useful in cases where a domain has more than one mail exchanger. A lower number denotes a higher priority, and mail will attempt to deliver through the highest priority exchanger before moving to lower priority exchangers. The default value is `10`.
+* `refresh` - (Optional, integer) The duration, expressed in seconds, that a secondary name server waits to check the domain's primary name server for a new copy of a DNS zone. If a zone file has changed, the secondary DNS server updates its copy of the zone to match the primary DNS server's zone.
+* `responsible_person` - (Required, string) The email address of the person responsible for a domain. Replace the `@` symbol in the address with a `.`. For example: root@example.org would be expressed as `root.example.org.`.
+* `retry` - (Optional, integer) The duration, expressed in seconds, that the domain's primary name server (or servers) waits before attempting to refresh the domain's zone with the secondary name server. A failed attempt to refresh by a secondary name server triggers the retry action.
+* `ttl` - (Required, integer) The time to live (TTL) duration, expressed in seconds, of a resource record. A name server uses TTL to determine how long to cache a resource record. An SOA record's TTL value defines the domain's overall TTL.
+* `type` - (Required, string) The type of domain resource record. Accepted values are as follows:
+    * `a` for address records
+    * `aaaa` for address records
+    * `cname` for canonical name records
+    * `mx` for mail exchanger records
+    * `ptr` for pointer records in reverse domains
+    * `spf` for sender policy framework records
+    * `srv` for service records
+* `txt` - (Optional, string) Used for text records.
+* `service` - (`SRV` records only, string) The symbolic name of the desired service.
+* `protocol` - (`SRV` records only, string) The protocol of the desired service. This is usually TCP or UDP.
+* `port` - (`SVR` records only, integer) The TCP or UDP port on which the service will be found.
+* `priority` - (`SVR` records only, integer) The priority of the target host. The lowest numerical value is given the highest priority.
+* `weight` - (`SVR` records only, integer) A relative weight for records that have the same priority.

```

Comment:
Are the above 5 SRV-records-only arguments required or optional for SRV records? @vkalangu

Reply:
Required

Reply:
Looks like we missed `SVR` to `SRV` change.